django-stubs/forms/boundfield.pyi

```diff
@@ -33,6 +34,9 @@ class BoundField:
     def __getitem__(self, idx: slice) -> list[BoundWidget]: ...
     @property
     def errors(self) -> ErrorList: ...
+    @property
+    def template_name(self) -> str: ...
+    def get_context(self) -> dict[str, Any]: ...
```

Comment:
We can now remove this line, because it is define here: https://github.com/typeddjango/django-stubs/blob/4a7619f7ce1e43f09299b02ae62f23e79766442e/django-stubs/forms/utils.pyi#L22
```suggestion
```

Reply:
I'm starting to see a little bit about the flow. :)